src/app/content/html/html-editor/html-editor.component.ts

```diff
@@ -234,11 +234,19 @@ export class HtmlEditorComponent implements OnInit, OnChanges {
     const path =
       element.path ?? component.pathNames.createFromBreadcrumbs(element);
 
+    // Need to do this for NHS England, local paths should be used so that links can be maintained within the context of the branches
+    // they live under
+    const pathElements = this.pathNames.parse(path);
+    const pathElementsWithoutBranch = this.pathNames.removeVersionOrBranchName(
+      pathElements
+    );
+    const pathWithoutBranch = this.pathNames.build(pathElementsWithoutBranch);
+
     // The href will be the Mauro item path. This is not a valid URL, but the HtmlParserService will
     // automatically look for these paths in the content and rewrite them to correct URLs before
     // rendering for display
     const html = editor.create.fromHTML(
```

Comment:
the link is fine here all works, its just after it gets borked

Reply:
The link is not meant to work whilst you're still in the editor, but when you save the changes and go back to the item detail view, the links will work because they are automatically converted.

Reply:
ahh I see, please ignore that then. I meant that when the link is created it is not added to the description at all if the node line crashes out